src/settings/Settings.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import Toggle from './Toggle';
+import AddTimerSchema from './AddTimerSchema';
+
+export default function Settings() {
+  const [isAutoStartTimer, setIsAutoStartTimer] = useState(() => false);
+  const [isAutoStartBreak, setIsAutoStartBreak] = useState(() => false);
```

Comment:
Why `() => false`?

Reply:
It was in progress, I stopped working due to mentioned reasons.